lib/jekyll/convertible.rb

```diff
@@ -72,24 +72,6 @@ def validate_permalink!(filename)
       end
     end
 
-    # Transform the contents based on the content type.
-    #
-    # Returns the transformed contents.
-    def transform
```

Comment:
We can't remove these from Convertible until Jekyll 4 (when we'll remove the whole module). These could proxy out to a `Jekyll::Renderer` instance created in a private helper method.